dqc_us_rules/dqc_us_0005.py

```diff
@@ -7,23 +7,39 @@
 from arelle.XmlUtil import dateunionValue
 
 LEGALENTITYAXIS_DEFAULT = ''
-_dei_pattern = re.compile(r'^http://xbrl\.((sec\.gov)|(us))/dei/\d{4}-\d{2}-\d{2}')
+_dei_pattern = \
```

Comment:
This seems very odd and unnecessary, do we have some odd definition to require this line continuation?